### PR TITLE
Completed alert messages if user tries to edit/delete others' posts

### PR DESCRIPTION
**Pull request by melvinlau**

### CODE REVIEW

**Comment by melvinlau** [comment]
Selenium webdriver was used with Capybara in order to test each alert message, which describe an error when the user attempts an unauthorised operation. All tests passed locally with Firefox providing visuals during the tests. Commits 07bb8fa to 3a98c00 should contain these in working order.

However, I am unable to set up Travis CI to replicate this. Therefore I'm reverting back to tests without using Selenium web driver. This means Capybara cannot evaluate alerts/tooltips and the test expectations will have to be based on presence of other permanent page content.

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -13,12 +13,15 @@
     <%= button_to "Edit", edit_post_path(post.id), :method => :get if (Time.now - post.created_at) / 60 <= 10%>
     <%= button_to "Delete", post_path(post.id), :method => :delete,
     data: {confirm: 'Are you sure you want to delete this post?'} %>
+  <% else %>
+    <%= button_to "Edit", posts_url, method: :get, data: { confirm: 'Permission denied' } if (Time.now - post.created_at) / 60 <= 10%>
+    <%= button_to "Delete", posts_url, method: :get, data: { confirm: 'Permission denied' } %>
   <% end %>
 
   <%= button_to "Buzz", like_post_path(post.id), :method => :put %>
```
Comment (JDtheGeek) [inline app/views/posts/index.html.erb]: Still has references to buzz

File: spec/features/user_can_edit_posts_spec.rb [hunk 2]
```diff
@@ -37,20 +37,8 @@
     create_user_two
     login_user_two
     visit '/posts'
-    expect(page).to have_no_button("Edit")
-  end
-
-  scenario "User cannot delete another user's post" do
```
Comment (JDtheGeek) [inline spec/features/user_can_edit_posts_spec.rb]: Is this meant to be deleted?

Comment (melvinlau) [reply spec/features/user_can_edit_posts_spec.rb]: This was moved to user_can_delete_posts_spec

**Pull request merged**